pkg/v11/resource/chartoperator/error.go

```diff
@@ -11,6 +11,15 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+var notReadyError = &microerror.Error{
+	Kind: "notReadyError",
+}
+
+// IsNotReady asserts notReadyError.
+func IsNotReady(err error) bool {
+	return microerror.Cause(err) == notReadyError
+}
```

Comment:
Neither of the existing errors are a good fit. I don't think execution failed makes sense because the error is temporary until the deployment is ready.